.github/workflows/scripts/update-eventuals-git-repository.sh

```diff
@@ -0,0 +1,77 @@
+# !/bin/bash
+
+# File passed to this script as an argument in which we gonna
+# update 'git_repository' bazel rule for eventuals 
+# (WORKSPACE.bazel usually).
+file=$1
+
+# The last commit for eventuals.
+commit=$2
+
+# The latest shallow_since for eventuals.
+shallow_since=$3
+
+if [[ ! -f "$file" ]]; then
+    echo "$file doesn't exist"
+    exit 1
+fi
+
+if [[ ${#commit} == 0 ]]; then
+    echo "Incorrect commit value."
+    exit 1
+fi
+
+if [[ ${#shallow_since} == 0 ]]; then
+    echo "Incorrect shallow_since value."
+    exit 1
+fi
+
+# This flag indicates whether or not we process correct
+# 'git_repository' rule for eventuals in current $file.
+git_repository_needed=False
+
+# Flags to help understand whether or not all updates
+# (commit and shallow_since) are done.
+is_commit_replaced=False
+is_shallow_replaced=False
+
+while IFS= read -r line
+do
```

Comment:
Let's keep the formatting consistent and we put things like `then` on the same line as `if`:

```suggestion
while IFS= read -r line; do
```